guides/release/components/looping-through-lists.md

```diff
@@ -279,7 +279,7 @@ import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
 import { A } from '@ember/array';
 
-export default class extends Component {
+export default class Messages extends Component {
```

Comment:
Seems good but we should be consistent with the class naming convention. For now, lets go with:

```suggestion
export default class MessagesComponent extends Component {
```